.env.sample

```
@@ -24,6 +24,7 @@ OTEL_PYTHON_EXCLUDED_URLS=".*health-check,.*__reload__,.*static,.*media,.*__debu
 PYTHONDONTWRITEBYTECODE="1"
 PYTHONPATH="."
 PYTHON_MAX_THREADS="4"
+REQUESTS_VERIFY_SSL="true"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)